### DISCUSSION TITLE
Use HTML in entries

### DISCUSSION THREAD

**Discussion by julesjanssen** [vdhicts/keepachangelog-parser#6] [Ideas]
First of all: thanks for open sourcing this lib.

I'd like to use bits of HTML in the entries of my changelog through markdown. Example:

`- Updated **some library** to fix [this nasty bug](https://link-to-bug/)`

Currently, the entries are returned as plain text (after being parsed as rich markdown results). Would you be open to a PR that returns HTML results per entry?
Or perhaps one that turns entries into Entry objects with a text & html method? The `__toString` on this object could return the text version, which could potentially make this a backwards compatible update.

**Comment by dvdheiden** [comment]
Hi @julesjanssen!

As long as the keepachangelog standard allows it, I allow it in this package. So a PR is absolutely welcome for this functionality and I actually think it would be a good idea to support the use case you mentioned in your example.

**Comment by julesjanssen** [comment]
See https://github.com/vdhicts/keepachangelog-parser/pull/7.

**Reply by dvdheiden** [reply]
Nice, thank you for your contribution! I will take a look at it.

**Comment by dvdheiden** [comment]
Just released this feature in v2.1.0! Thank you for your work!

**Reply by julesjanssen** [reply]
Thanks.
